test/RelayClient.test.ts

```diff
@@ -40,11 +40,123 @@ const FAKE_TX_COUNT = 456;
 const FAKE_CHAIN_ID = 33;
 const FAKE_SIGNATURE = 'FAKE_SIGNATURE';
 
+const createRandomAddress = () => Wallet.createRandom().address;
+
 describe('RelayClient', function () {
   afterEach(function () {
     sandbox.restore();
   });
 
+  describe('constructor', function () {
+    it('should store provider', function () {
```

Comment:
I don't get the idea of this test.  I would prefer check if getDefaultProvider() and getEnvelopingConfig() were called instead of setting the provider manually. I'll be asking you about this because perhaps I'm missing something that I can learn.